app/repository/RiderRepositoryImpl.java

```diff
@@ -29,69 +25,80 @@ public RiderRepositoryImpl(JPAApi jpaApi, DatabaseExecutionContext databaseExecu
     }
 
     @Override
-    public CompletionStage<JsonNode> getAllRiders() {
-        return supplyAsync(() -> wrap (this::getAllRiders), databaseExecutionContext);
+    public List<Rider> getAllRiders() {
+        return wrap(this::getAllRiders);
     }
 
-    private JsonNode getAllRiders(EntityManager em){
+    private List<Rider> getAllRiders(EntityManager em){
         List<Rider> riders = em.createQuery("select r from Rider r", Rider.class).getResultList();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "riders". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)